package.json

```diff
@@ -21,7 +21,7 @@
     }
   },
   "lint-staged": {
-    "src/client/**/*.js": "lint"
+    "src/client/**/*.{js,jsx}": "npm run lint"
```

Comment:
This is also trivial and depends on you.
But, my suggestion here is, you can say `Support linting JSX in lint-staged` so that you can know much quicker when you see the commit message next time.